### PR TITLE
chore: improve warning messages and titles

### PR DESCRIPTION
**Pull request by RanGojo**
# Summary

In this PR, I've updated warning messages in widget.

Fixes RF-1871

# How did you test this change?

By checking related section specially adding experimental chain.
Also, we shouldn't use Blockchain anymore.

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: widget/embedded/src/components/Slippage/SlippageTooltipContent.tsx [hunk 1]
```diff
@@ -9,13 +9,13 @@ export function SlippageTooltipContent() {
...
-          'Your transaction will be reverted if the price changes unfavorably by more than this percentage'
+          'Your transaction will be reverted if the price changes unfavorably by more than this percentage.'
         )}
         <br />
         <br />
-        <b>{i18n.t('Warning')}</b>:
+        <b>{i18n.t('Warning')}</b>:&nbsp;
         {i18n.t(
-          'This setting is applied per step (e.g. 1Inch, Thorchain, etc) which means only the step will be reverted, not the whole transaction.'
+          'This setting is applied to each individual step (e.g. 1Inch, Thorchain, etc.), meaning only that specific step will be reverted, not the entire route.'
```
Comment (RyukTheCoder) [inline widget/embedded/src/components/Slippage/SlippageTooltipContent.tsx:18]:
```suggestion
          'This setting is applied to each step (e.g. 1Inch, Thorchain, etc.), meaning only that specific step will be reverted, not the entire route.'
```
Redundant

File: widget/embedded/src/pages/LanguagePage.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export function LanguagePage() {
         <Alert
           type="warning"
           variant="alarm"
-          title="Important Notice: Machine translations are in use"
+          title="Warning: We are using machine translation, so the translations may not be very accurate."
```
Comment (RyukTheCoder) [inline widget/embedded/src/pages/LanguagePage.tsx:49]:
```suggestion
          title="Warning: We are using machine translation, so the translations may be inaccurate.."
```

**Review by RyukTheCoder — CHANGES_REQUESTED** [review]
Tnx, I just added some minor comments.

File: widget/embedded/src/pages/LanguagePage.tsx (hunk 2 above)
Comment (RanGojo) [reply widget/embedded/src/pages/LanguagePage.tsx]: fixed

File: widget/embedded/src/components/Slippage/SlippageTooltipContent.tsx (hunk 1 above)
Comment (RanGojo) [reply widget/embedded/src/components/Slippage/SlippageTooltipContent.tsx]: done

**Review by RyukTheCoder — APPROVED** [review]
LGTM

**Pull request merged**